eq-author/src/App/questionConfirmation/Design/ConfirmationOption.js

```diff
@@ -57,6 +71,9 @@ ConfirmationOption.propTypes = {
   label: PropTypes.string.isRequired,
   onChange: PropTypes.func.isRequired,
   onUpdate: PropTypes.func.isRequired,
+  getValidationError: PropTypes.func.isRequired,
 };
 
-export default ConfirmationOption;
+export default flowRight(withValidationError(CONFIRMATION_OPTION))(
```

Comment:
Could we do `withValidationError(CONFIRMATION_OPTION)(UnwrappedConfirmationOption)` just to save importing flowRight?

Reply:
Changed